src/structures/GuildMember.js

```diff
@@ -346,6 +346,30 @@ class GuildMember extends Base {
     return this.guild.members.fetch({ user: this.id, cache: true, force });
   }
 
+  /**
+   * Whether this guild member equals another guild member. It compares all properties, so for most
+   * comparison it is advisiable to just compare `member.id === member2.id` as it is significantly faster
+   * and is often what most users need.
+   * @param {GuildMember} member the member to compare with
+   * @returns {boolean}
+   */
+  equals(member) {
+    return (
+      member &&
+      member instanceof this.constructor &&
```

Comment:
The truthy check is useless as the `instanceof` will always return `false` for nullish values.
```suggestion
      member instanceof this.constructor &&
```